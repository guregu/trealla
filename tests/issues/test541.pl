:- initialization(main).
:- op(0,yfx,-).
:- op(0,fy,--).

main :-
	X = -(---(_A),-_B),
	Y = -(- --(_A),-_B),
	write([X,Y]), nl.
