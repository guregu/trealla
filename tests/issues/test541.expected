[-(---(_5),-_6),-(- --(_5),-_6)]
